src/jmh/java/com/datadoghq/sketch/ddsketch/benchmarks/AcceptValue.java

```diff
@@ -35,6 +35,9 @@
     public void init() {
         this.sketch = sketchOption.create(relativeAccuracy);
         this.data = new long[1 << logCount];
+        for (int i = 0; i < data.length; ++i) {
+            data[i] = unit.toNanos(Math.abs(Math.round(generator.nextValue())));
```

Comment:
Why do we use `Math.abs`? The sketches should handle negative values, and I believe that if we really want to restrict to non-negative values, we should use appropriate generators/distributions rather than artificially folding the distribution of values here. What do you think?

Reply:
This is a carry over from when `DDSketch` couldn't take negatives, let me update it

Reply:
@CharlesMasson I've updated this now